### DISCUSSION TITLE
Another problem with Basset

### DISCUSSION THREAD

**Discussion by blondie63** [Laravel-Backpack/community-forum#1065] [Q&A (Help)] [answered]
I've this filesystem.php
```
<?php

return [

    /*
    |--------------------------------------------------------------------------
    | Default Filesystem Disk
    |--------------------------------------------------------------------------
    |
    | Here you may specify the default filesystem disk that should be used
    | by the framework. The "local" disk, as well as a variety of cloud
    | based disks are available to your application. Just store away!
    |
    */

    'default' => env('FILESYSTEM_DISK', 'local'),

    /*
    |--------------------------------------------------------------------------
    | Filesystem Disks
    |--------------------------------------------------------------------------
    |
    | Here you may configure as many filesystem "disks" as you wish, and you
    | may even configure multiple disks of the same driver. Defaults have
    | been set up for each driver as an example of the required values.
    |
    | Supported Drivers: "local", "ftp", "sftp", "s3"
    |
    */

    'disks' => [

        'local' => [
            'driver' => 'local',
            'root' => storage_path('app'),
            'throw' => false,
        ],

        'public' => [
            'driver' => 'local',
            'root' => storage_path('app/public'),
            'url' => env('APP_URL').'/storage',
            'visibility' => 'public',
            'throw' => false,
        ],


        'basset' => [
            'driver' => 'local',
            'root' => public_path(''),
            'url' => '/',
            'visibility' => 'public',
            'throw' => false,
        ],

        'data' => [
            'driver' => 'local',
            'root' => public_path('storage'),
            'url' => '/storage',
            'visibility' => 'public',
            'throw' => true,
        ],

    ],

    /*
    |--------------------------------------------------------------------------
    | Symbolic Links
    |--------------------------------------------------------------------------
    |
    | Here you may configure the symbolic links that will be created when the
    | `storage:link` Artisan command is executed. The array keys should be
    | the locations of the links and the values should be their targets.
    |
    */
];
```

on my .env i've set BASSET_DISK=basset

on server i run these cmds:
<img width="727" alt="basset_cmds" src="https://github.com/Laravel-Backpack/community-forum/assets/1261851/020d3230-695e-4ca9-9b4e-b1435a0016a0">

But my app does'nt works with this error:
<img width="1561" alt="Screenshot 2024-07-03 alle 11 54 22" src="https://github.com/Laravel-Backpack/community-forum/assets/1261851/cbd014de-3dae-4c7b-8143-91ed688adc8a">

Given that my deployment system already creates and uses the symlink storage which points to a folder external to the app to keep user files persistent, I chose to use a specific disk for basset and it worked for a few days but not anymore, strangely!
It's strange that "php artisan basset:check" says everything is ok when in reality it doesn't work..

Can anyone help me understand where I'm wrong?

**Comment by blondie63** [comment]
@pxpm Honestly, in a production environment, for Bassett to work optimally, where should the basset disk point and how should it be configured?

**Comment by emjayess** [comment, marked as answer]
@blondie63 that page in your screen capture looks to be the Laravel front, which is linking off to the backpack /admin login page; and, further, it looks like the laravel app is employing react on the front-end, in some capacity? Those failed template rendering issues are reminiscent of something mis-configured to create harmony among blade and react/js templating directives; and/or a failed step in the build step for that javascript front-end.

**Reply by blondie63** [reply]
After further research I think I have discovered the problem! I had some calls that used asset() instead of basset() but I'm monitoring the situation..

**Comment by pxpm** [comment]
@emjayess I think you are correct or somewhat in the right direction, thanks for helping out 🙏 

@blondie63 let me know if you still need my help here, but I don't think I would have much to add. I don't see nothing wrong with your setup. 

Cheers

**Reply by blondie63** [reply]
@pxpm Thanks, no need more
Regards
